src/foam/strategy/BasicStrategizer.js

```diff
@@ -54,11 +57,23 @@ foam.CLASS({
           predicate = (Predicate) strategyPredicate;
         }
 
-        DAO strategyDAO = ((DAO) getStrategyDAO()).inX(x);
-        List refs = ((ArraySink) strategyDAO
+        AuthService auth = (AuthService) x.get("auth");
```

Comment:
Why do this here instead of just turning on the standard authorizer on the strategyDAO?

Reply:
Yes, that should be the correct approach.  I'll investigate as it's not working - this is the default for EasyDAO and it's not explicitly disabled for strategyDAO. 
Ah, the client is going through a service, but still this should be working.